tools/firmware.toit

```diff
@@ -506,18 +506,17 @@ flash parsed/cli.Parsed -> none:
   firmware_bin := extract_binary envelope --config_encoded=config_encoded
   binary := Esp32Binary firmware_bin
 
-  separator := ?
   bin_extension := ?
+  bin_name := program_name
   if platform == PLATFORM_WINDOWS:
-    separator = "\\"
+    bin_name.replace --all "\\" "/"
```

Comment:
I'm surprised by this. Is replace in-place?

Reply:
It's not...
I moved the code around to make it nicer, and the forgot to assign it...
done.